src/ansys/geometry/core/_grpc/_services/v1/conversions.py

```diff
@@ -1335,14 +1370,20 @@ def from_angle_to_grpc_quantity(input: "Measurement") -> GRPCQuantity:
     Parameters
     ----------
     input : Measurement
-        Source measurement data.
+        Source measurement data (Measurement object or pint Quantity).
 
     Returns
     -------
     GRPCQuantity
         Converted gRPC quantity.
     """
-    return GRPCQuantity(value_in_geometry_units=input.value.m_as(DEFAULT_UNITS.SERVER_ANGLE))
+    # Handle both Measurement objects (which have .value attribute) and raw pint Quantities
+    if hasattr(input, "value"):
+        # Measurement object
+        return GRPCQuantity(value_in_geometry_units=input.value.m_as(DEFAULT_UNITS.SERVER_ANGLE))
+    else:
+        # Raw pint Quantity
+        return GRPCQuantity(value_in_geometry_units=input.m_as(DEFAULT_UNITS.SERVER_ANGLE))
```

Comment:
Rather than using "hasattr" I'd recommend to use "isinstance" - that will make things safer (for example, imagine that the pint.Quantity object decides to include a "value" property that contains something totally different... that would directly break our code I believe)